UnityProject/Assets/Scripts/Managers/EventManager.cs

```diff
@@ -31,7 +31,8 @@ public enum EVENT
 	PlayerRejoined,
 	PreRoundStarted,
 	MatrixManagerInit,
-	BlobSpawned
+	BlobSpawned,
+	ItemSold
```

Comment:
remove this value from the enum, we're not using it anymore and should only use the c# event.

Reply:
Good point.